ansible/vars/topo_ciscovs-7nodes.yml

```diff
@@ -0,0 +1,218 @@
+topology:
+  host_interfaces:
+    - 0
+    - 1
+    - 2
+    - 3
+    - 4
+    - 5
+    - 6
+    - 7
+    - 8
+    - 9
+    - 10
+    - 11
+    - 12
+    - 13
+    - 14
+    - 15
+    - 21
+    - 22
+    - 23
+    - 24
+    - 25
+    - 26
+    - 27
+  disabled_host_interfaces:
+    - 0
+    - 25
+    - 26
+    - 27
+  VMs:
+    PE1:
+      vlans:
+        - 28
+      vm_offset: 0
+    PE2:
+      vlans:
+        - 29
+      vm_offset: 1
+    PE3:
+      vlans:
+        - 30
+      vm_offset: 2
+    P3:
+      vlans:
+        - 31
+      vm_offset: 3
+    P2:
+      vlans:
+        - 16
+      vm_offset: 4
+    P4:
+      vlans:
+        - 17
+      vm_offset: 5
+  VM_LINKs:
+    LINK1:
+      start_vm_offset: 0
+      start_vm_port_idx: 1
+      end_vm_offset: 3
+      end_vm_port_idx: 1
+    LINK2:
+      start_vm_offset: 1
+      start_vm_port_idx: 2
+      end_vm_offset: 3
+      end_vm_port_idx: 2
+    LINK3:
+      start_vm_offset: 3
+      start_vm_port_idx: 3
+      end_vm_offset: 4
+      end_vm_port_idx: 1
+    LINK4:
+      start_vm_offset: 3
+      start_vm_port_idx: 4
+      end_vm_offset: 5
+      end_vm_port_idx: 1
+    LINK5:
```

Comment:
Is there a better way to name the links to reflect their purpose? A sequence number suffix is not self explanatory enough.

Reply:
Changed to use two end point nodes' name combinations as link names.

Reply:
VM_LINKs:
    PE1P3:
      start_vm_offset: 0
      start_vm_port_idx: 1
      end_vm_offset: 3
      end_vm_port_idx: 1
    PE2P3:
      start_vm_offset: 1
      start_vm_port_idx: 2
      end_vm_offset: 3
      end_vm_port_idx: 2
      
      https://github.com/eddieruan-alibaba/sonic-mgmt/blob/eruan-testcasemaster/ansible/vars/topo_ciscovs-7nodes.yml#L56C1-L66C25

Reply:
Is the change pushed to PR branch?

Reply:
Same as above. It got cherry-picked now.